File: ApiSuvesaPos/Negocio/Modelo/cxc/ventas/ventas.vb
Namespace Modelo
    Public Class ventas
        Public Property Id As Long
        Public Property Num_Factura As Decimal
        Public Property Tipo As String
        Public Property Cod_Cliente As Long
        Public Property Nombre_Cliente As String
        Public Property Orden As String
        Public Property Cedula_Usuario As String
        Public Property Pago_Comision As Boolean
        Public Property SubTotal As Decimal
        Public Property Descuento As Decimal
        Public Property Imp_Venta As Decimal
        Public Property Total As Decimal
        Public Property Fecha As Date
        Public Property Vence As Date
        Public Property Cod_Encargado_Compra As String
        Public Property Contabilizado As Boolean
        Public Property AsientoVenta As Long
        Public Property ContabilizadoCVenta As Boolean
        Public Property AsientoCosto As Long
        Public Property Anulado As Boolean
        Public Property PagoImpuesto As Integer
        Public Property FacturaCancelado As Boolean
        Public Property Num_Apertura As Long
        Public Property Entregado As Boolean
        Public Property Cod_Moneda As Integer
        Public Property Moneda_Nombre As String
        Public Property Direccion As String
        Public Property Telefono As String
        Public Property SubTotalGravada As Decimal
        Public Property SubTotalExento As Decimal
        Public Property Transporte As Decimal
        Public Property Tipo_Cambio As Decimal
        Public Property Observaciones As String
        Public Property Exonerar As Boolean
        Public Property Taller As Boolean
        Public Property Mascotas As Boolean
        Public Property Num_Caja As Long
        Public Property agente As Boolean
        Public Property cod_agente As Long
        Public Property apartado As Long
        Public Property EnviadoMH As Boolean
        Public Property Cedula As String
        Public Property ClaveMH As String
        Public Property EstadoMH As String
        Public Property TipoCedula As String
        Public Property ConsecutivoMH As String
        Public Property excluirCxC As Boolean
        Public Property Id_Adelanto As Integer
        Public Property Reenviar As Boolean
        Public Property NumeroReenvio As Integer
        Public Property ClaveReenvio As String
        Public Property ConsecutivoReenvio As String
        Public Property Archivada As Boolean
        Public Property UsuarioArchivo As String
        Public Property FechaArchivo As Date
        Public Property Frecuente As Boolean
        Public Property Id_FacturaRemplaza As Integer
        Public Property Renta As Boolean
        Public Property Etapa As Integer
        Public Property FechaEtapa As Date
        Public Property EnProcesoIncobrable As Boolean
        Public Property Ventas_Detalle As New List(Of Modelo.ventas_detalle)

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/ventas/ventas.cs
using System;
using System.Collections.Generic;

namespace Modelo
{
    public class ventas
    {
        public long Id { get; set; }
        public decimal Num_Factura { get; set; }
        public string Tipo { get; set; }
        public long Cod_Cliente { get; set; }
        public string Nombre_Cliente { get; set; }
        public string Orden { get; set; }
        public string Cedula_Usuario { get; set; }
        public bool Pago_Comision { get; set; }
        public decimal SubTotal { get; set; }
        public decimal Descuento { get; set; }
        public decimal Imp_Venta { get; set; }
        public decimal Total { get; set; }
        public DateTime Fecha { get; set; }
        public DateTime Vence { get; set; }
        public string Cod_Encargado_Compra { get; set; }
        public bool Contabilizado { get; set; }
        public long AsientoVenta { get; set; }
        public bool ContabilizadoCVenta { get; set; }
        public long AsientoCosto { get; set; }
        public bool Anulado { get; set; }
        public int PagoImpuesto { get; set; }
        public bool FacturaCancelado { get; set; }
        public long Num_Apertura { get; set; }
        public bool Entregado { get; set; }
        public int Cod_Moneda { get; set; }
        public string Moneda_Nombre { get; set; }
        public string Direccion { get; set; }
        public string Telefono { get; set; }
        public decimal SubTotalGravada { get; set; }
        public decimal SubTotalExento { get; set; }
        public decimal Transporte { get; set; }
        public decimal Tipo_Cambio { get; set; }
        public string Observaciones { get; set; }
        public bool Exonerar { get; set; }
        public bool Taller { get; set; }
        public bool Mascotas { get; set; }
        public long Num_Caja { get; set; }
        public bool agente { get; set; }
        public long cod_agente { get; set; }
        public long apartado { get; set; }
        public bool EnviadoMH { get; set; }
        public string Cedula { get; set; }
        public string ClaveMH { get; set; }
        public string EstadoMH { get; set; }
        public string TipoCedula { get; set; }
        public string ConsecutivoMH { get; set; }
        public bool excluirCxC { get; set; }
        public int Id_Adelanto { get; set; }
        public bool Reenviar { get; set; }
        public int NumeroReenvio { get; set; }
        public string ClaveReenvio { get; set; }
        public string ConsecutivoReenvio { get; set; }
        public bool Archivada { get; set; }
        public string UsuarioArchivo { get; set; }
        public DateTime FechaArchivo { get; set; }
        public bool Frecuente { get; set; }
        public int Id_FacturaRemplaza { get; set; }
        public bool Renta { get; set; }
        public int Etapa { get; set; }
        public DateTime FechaEtapa { get; set; }
        public bool EnProcesoIncobrable { get; set; }
        public List<Modelo.ventas_detalle> Ventas_Detalle { get; set; } = new List<Modelo.ventas_detalle>();
    }
}
